llm_sandbox/docker.py

```
@@ -180,6 +187,27 @@ def __init__(
             else:
                 existing_mounts.append(mounts)
 
+    def _container_security_defaults(self) -> dict[str, Any]:
+        """Return backend container security defaults for the configured profile."""
+        if self.config.security_profile != RuntimeSecurityProfile.STRICT:
+            return {"user": "root"}
+
+        tmpfs = {"/tmp": "rw,nosuid,size=256m,mode=1777"}
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)